diff -Nrup a/meson.build b/meson.build
--- a/meson.build	2019-03-11 21:25:53.000000000 +0100
+++ b/meson.build	2019-03-14 08:53:49.204772811 +0100
@@ -150,7 +150,7 @@ pc_conf.set('prefix', get_option('prefix
 pc_conf.set('exec_prefix', '${prefix}')
 pc_conf.set('libdir', join_paths(get_option('prefix'), get_option('libdir')))
 pc_conf.set('includedir', join_paths('${prefix}', get_option('includedir')))
-pc_conf.set('VERSION', '"@0@"'.format(meson.project_version()))
+pc_conf.set('VERSION', '@0@'.format(meson.project_version()))
 pc_conf.set('RYGEL_PLUGIN_DIR', rygel_plugindir)
 pc_conf.set('RYGEL_MEDIA_ENGINE_DIR', rygel_enginedir)
 pc_conf.set('RYGEL_PLUGIN_LINKER_FLAGS', '')
