diff -Nrup a/meson.build b/meson.build
--- a/meson.build	2019-03-11 21:25:53.000000000 +0100
+++ b/meson.build	2019-03-12 05:35:06.765354039 +0100
@@ -24,7 +24,7 @@ if not get_option('uninstalled')
     rygel_plugindir = join_paths(rygel_libdir, 'rygel-2.6', 'plugins')
     rygel_enginedir = join_paths(rygel_libdir, 'rygel-2.6', 'engines')
     rygel_presetdir = join_paths(rygel_datadir, 'rygel', 'presets')
-    rygel_icondir = join_paths(rygel_datadir, 'icons', 'hicolor')
+    rygel_icondir = join_paths(get_option('prefix'), get_option('datadir'), 'icons', 'hicolor')
     rygel_bigicondir = join_paths(rygel_datadir, 'rygel', 'icons', '120x120')
     rygel_smallicondir = join_paths(rygel_datadir, 'rygel', 'icons', '48x48')
 else
